tests/playwright/sanity/assets/dev/js/frontend/handlers/progress.test.ts

```diff
@@ -0,0 +1,25 @@
+import { expect, test } from '@playwright/test';
+import WpAdminPage from '../../../../../../pages/wp-admin-page';
+
+test.describe.only( 'ProgressBar test: @progress', () => {
```

Comment:
We don't have a tag `@progress`. We only create the tags for larger widgets with multiple tags.

```suggestion
test.describe( 'ProgressBar test:', () => {
```